README.md

```diff
@@ -294,6 +305,9 @@ After the trial period, termination notice has to be given in writing, both by y
 
 Once you have handed in your termination notice, you are expected to gradually decrease your workload until the day you leave. During this time you should ensure you close all open activities, or pass them on to a colleague. You will no longer accept new projects or work fully on current projects. This is to ensure that nothing is left unfinished and that a colleague can easily continue the work once you have left.
 
+#### Parachute
+We believe that the company is a platform where people can develop their talents. People are people also outside the company, and we think we should help people leave the company without to much stress. The Parachute is the option that the company gives to a colleague who is going to come back in 12 months at the same salary. When leaving we will ask all the peers if to provide the parachute with and if the 10% is favorable add the parachute letter in case you want to come back within 12 months.
```

Comment:
We believe that the company is a platform in which everyone can develop their talents. As we value our colleagues inside the company, we also value them when they wish to leave, so we think we should help people leave without to much stress. Our Parachute policy gives everyone the option to return to the company within 12 months of leaving, and receive the same salary. When someone decides to leave we will ask all colleagues to agree to the parachute policy option: as long as 10% of the company workforce is favorable the person leaving will receive a letter in which this option is specified and agreed.